src/shapes/text.class.js

```diff
@@ -400,13 +400,14 @@
       }
       this._splitText();
       this._clearCache();
+      var textHeight = this.calcTextHeight();
       if (this.path) {
-        this.width = this.path.width;
-        this.height = this.path.height;
+        this.width = this.path.width + textHeight;
+        this.height = this.path.height + textHeight;
```

Comment:
This works because you make a single line of text.
It grows if is more than one line. I would use just the function that calculate the lineHeight ( the one called from calcTextHeight)

Reply:
right. I can fix that